src/Raven.Client/ServerWide/Operations/ConnectionStrings/GetConnectionStringsOperation.cs

```diff
@@ -13,30 +13,47 @@ public class GetConnectionStringsOperation: IServerOperation<GetConnectionString
     {
         private readonly string _databaseName;
 
-        public GetConnectionStringsOperation(string databaseName)
+        private readonly string _connectionStringName;
+
+        private readonly ConnectionStringType _type;
+
+        public GetConnectionStringsOperation(string databaseName, string connectionStringName=null, ConnectionStringType type = ConnectionStringType.None)
         {
             _databaseName = databaseName;
+            _connectionStringName = connectionStringName;
+            _type = type;
```

Comment:
Maybe make it nullable and remove ConnectionStringType.None from enum completely ?

Reply:
All enums should have `None`